<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstCh</class>
 <widget class="QWidget" name="InstCh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>77</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(82, 82, 82);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnMute">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>17</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSolo">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>17</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="FaderSlider" name="slider">
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>121</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LEDVu" name="vuBar">
       <property name="minimumSize">
        <size>
         <width>14</width>
         <height>121</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbImage">
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbName">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LEDVu</class>
   <extends>QFrame</extends>
   <header>LEDVu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FaderSlider</class>
   <extends>QFrame</extends>
   <header>FaderSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
